routes/notifications.js

```diff
@@ -0,0 +1,161 @@
+const express = require("express");
+const router = express.Router();
+const Notification = require("../models/Notification");
+const joi = require("@hapi/joi");
+
+const notificationValidation = joi.object({
+  userId: joi.string().required(),
+  type: joi.string().required(),
+  isRead: joi.boolean(),
+  date: joi.date(),
+  content: joi.string().required().max(200),
+});
+
+const updateNotificationValidation = joi.object({
+  isRead: joi.boolean().required(),
+});
+
+// http://localhost/api/notifications ///////////////////////////////////////////////////////
+// GET
+// Returns all notifications
+
+router.get("/", async (req, res) => {
```

Comment:
I can't imagine a single use case for this endpoint. Let's try to keep the code clean by only including endpoints that we need

Reply:
true will delete